### DISCUSSION TITLE
Discussion on tcp plugin to achieve TPS flow limitation

### DISCUSSION THREAD

**Discussion by xiupengrong** [Kong/kong#8469] [Help]
hi，all:

Kong for tcp protocol proxy, can implement the TPS flow limit function? Or can Kong implement the access stage in TCP plug-in? Please give me some direction and reference, thanks!

**Comment by xiupengrong** [comment]
Does everyone have a need to limit the flow based on the tcp protocol?

**Comment by fffonion** [comment]
@xiupengrong This is currently not supported, but we are open to PRs! The rate-limiting plugin is one you can start with.
